com.github.calo001.luna.json

```diff
@@ -0,0 +1,50 @@
+{
+    "app-id" : "com.github.calo001.luna",
+    "base" : "io.elementary.BaseApp",
+    "base-version" : "juno",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.30",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "com.github.calo001.luna",
+    "finish-args" : [
+        "--share=ipc", "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "build-options" : {
```

Comment:
Those are default now. You can remove them. They are used from freedesktop runtime 18.08 which is the base of the gnome & kde runtimes.

Reply:
Fixed :+1:

Reply:
You have also removed dconf settings permissions :/